lib/ansible/module_utils/aci.py

```diff
@@ -128,7 +137,9 @@ def define_method(self):
 
         # Handle deprecated method/action parameter
         if self.params['method']:
-            self.module.deprecate("Parameter 'method' or 'action' is deprecated, please use 'state' instead", '2.6')
+            # Deprecate only if state was a valid option (not for aci_rest)
+            if self.module.argument_spec('state', False):
```

Comment:
Because `state` isn't defined in `aci_argument_spec`, should not this done in `aci_tenant.py` ? I mean, `ACIModule` could be subclassed in `aci_tenant.py` and `define_method` could be overridden here.